sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -581,10 +580,10 @@ def send_event(self, event_data, **kwargs):
         :raises OperationTimeoutError: If the value specified by the timeout parameter elapses before the event can be
          sent in non-buffered mode or the events can be enqueued into the buffered in buffered mode.
         """
-        input_pid = kwargs.get("partition_id")
-        pid = input_pid or ALL_PARTITIONS
-        partition_key = kwargs.get("partition_key")
-        send_timeout = kwargs.get("timeout")
+        input_pid: str = cast(str, kwargs.get("partition_id"))
```

Comment:
something something expand kwargs. But also should we just `pop` here if we expect it?

Reply:
expanded kwargs so didn't pop